homeassistant/components/media_player/__init__.py

```diff
@@ -536,16 +536,19 @@ def supported_features(self):
             # If this platform is still using supported_media_commands, issue
```

Comment:
We could, I personally don't think it's really worth it. It's simple enough, and since it's temporary if it breaks later it was going to get removed anyway.